zwave_js/rootfs/usr/share/tempio/zwave_config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "logConfig": {
+        "enabled": true,
+        "level": 4,
+        "forceConsole": true
+    },
+    "storage": {
+        "driver": "FileSystem",
```

Comment:
having this here causes:
```
Unable to start driver TypeError: Cannot use 'in' operator to search for 'appendFile' in FileSystem
```

I was able to work around this by removing this line.